app/src/main/java/com/yeonae/chamelezone/view/course/CourseRegisterActivity.kt

```diff
@@ -46,13 +46,15 @@ class CourseRegisterActivity : AppCompatActivity(), CourseRegisterContract.View,
         toast("$tag")
         imageContainer.removeAllViews()
         uris.forEach { uri ->
-            val iv = LayoutInflater.from(this).inflate(
+            val rl = LayoutInflater.from(this).inflate(
                 R.layout.slider_item_image,
                 imageContainer,
                 false
-            ) as ImageView
-            imageContainer.addView(iv)
-            iv.glideImageSet(uri, image_item.measuredWidth, image_item.measuredHeight)
+            ) as RelativeLayout
+            imageContainer.addView(rl)
+            rl.image_item.run {
+                glideImageSet(uri, measuredWidth, measuredHeight)
+            }
```

Comment:
```suggestion
            rl.image_item.glideImageSet(uri, measuredWidth, measuredHeight)
```
한가지 작업만 하는데 run을 사용할 필요없을것 같아요

Reply:
rl.image_item.glideImageSet(uri, image_item.measuredWidth, image_item.measuredHeight)
이렇게 image_item을 계속 사용하는 것보다 run을 쓰는게 나을 것 같아서 사용했습니다